src/app/showcase/modal-showcase/modal-showcase.component.ts

```diff
@@ -19,17 +19,23 @@ export class ModalShowcaseComponent {
       inputValues: ['string'],
     },
     {
-      name: 'titleHorizontalAlignment',
-      description: 'The horizontal alignment of the title',
-      defaultValue: 'center',
-      inputValues: ['left', 'center'],
+      name: 'closeBtnPlacement',
```

Comment:
Consider renaming to:
```suggestion
      name: 'closeBtnPosition',
```